sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/MapsSearchAsyncClient.java

```diff
@@ -9,52 +9,17 @@
 import com.azure.core.exception.HttpResponseException;
 import com.azure.core.http.HttpPipeline;
 import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.ResponseBase;
 import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.models.GeoBoundingBox;
 import com.azure.core.models.GeoPosition;
 import com.azure.core.util.Context;
-import com.azure.core.util.FluxUtil;
-import com.azure.core.util.polling.DefaultPollingStrategy;
-import com.azure.core.util.polling.PollerFlux;
-import com.azure.core.util.serializer.TypeReference;
 import com.azure.maps.search.implementation.SearchesImpl;
-import com.azure.maps.search.implementation.helpers.BatchResponseSerializer;
-import com.azure.maps.search.implementation.helpers.Utility;
-import com.azure.maps.search.implementation.models.BatchRequest;
-import com.azure.maps.search.implementation.models.BatchRequestItem;
-import com.azure.maps.search.implementation.models.ErrorResponseException;
-import com.azure.maps.search.implementation.models.GeoJsonLineString;
-import com.azure.maps.search.implementation.models.GeoJsonObject;
-import com.azure.maps.search.implementation.models.JsonFormat;
-import com.azure.maps.search.implementation.models.ResponseFormat;
-import com.azure.maps.search.implementation.models.SearchAlongRouteRequest;
-import com.azure.maps.search.implementation.models.SearchInsideGeometryRequest;
-import com.azure.maps.search.models.BatchReverseSearchResult;
-import com.azure.maps.search.models.BatchSearchResult;
-import com.azure.maps.search.models.FuzzySearchOptions;
-import com.azure.maps.search.models.MapsPolygon;
-import com.azure.maps.search.models.PointOfInterestCategoryTreeResult;
-import com.azure.maps.search.models.ReverseSearchAddressOptions;
-import com.azure.maps.search.models.ReverseSearchAddressResult;
-import com.azure.maps.search.models.ReverseSearchCrossStreetAddressOptions;
-import com.azure.maps.search.models.ReverseSearchCrossStreetAddressResult;
-import com.azure.maps.search.models.SearchAddressOptions;
-import com.azure.maps.search.models.SearchAddressResult;
-import com.azure.maps.search.models.SearchAlongRouteOptions;
-import com.azure.maps.search.models.SearchInsideGeometryOptions;
-import com.azure.maps.search.models.SearchNearbyPointsOfInterestOptions;
-import com.azure.maps.search.models.SearchPointOfInterestCategoryOptions;
-import com.azure.maps.search.models.SearchPointOfInterestOptions;
-import com.azure.maps.search.models.SearchStructuredAddressOptions;
-import com.azure.maps.search.models.StructuredAddress;
+import com.azure.maps.search.implementation.models.*;
+import com.azure.maps.search.models.*;
```

Comment:
`*` imports will be flagged by linting. If using IntelliJ there is a setting to configure how many imports from the same package are required for a `*` import to be used, set that to 99 to make IntelliJ effectively never use `*` imports.